src/tests/conformance/query/peer_shard_info.rs

```diff
@@ -1,10 +1,21 @@
 //! Contains test with peer shard info queries.
 
+use tempfile::TempDir;
+
 use crate::{
-    protocol::codecs::binary::{BinaryMessage, Payload},
+    protocol::{
+        codecs::binary::{BinaryMessage, Payload},
+        proto::{TmGetPeerShardInfoV2, TmPublicKey},
+    },
+    setup::node::{Node, NodeType},
     tests::conformance::perform_response_test,
-    tools::config::TestConfig,
+    tools::{config::TestConfig, synth_node::SyntheticNode},
 };
+const KEY_TYPES: &[u8] = &[
+    0xED, // ed25519
+    0x02, // secp256k1
+    0x03, // secp256k1
```

Comment:
this is the same as `0x02`, it's probably not correct?

Reply:
Secp256k1 has two correct values.

Reply:
we should note how this is different (or not), otherwise this is unclear / suggests a mistake

Reply:
Done